treehouse/datetime.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime, timedelta
+
+
+def today(date_format: str) -> str:
```

Comment:
Maybe add a default format string here? The added value of this function is quite small.